### ISSUE TITLE
feature request: prevent open in callback

### ISSUE THREAD

**Issue by viglucci** [ssleptsov/ninja-keys#13]
Is there currently a way to prevent the menu from opening when the hotkey (`ctrl + k`) is invoked?

I have a scenario where the component which mounts the `ninja-key` element has an initial loading state, and I would like to be able to prevent the command menu from opening while in the loading state.

Thanks

**Comment by viglucci** [comment]
Thanks!

One quick question: will the `enabled/disabled` property also prevent the children hotkeys from invoking as well? I think if `enabled: false` is set, then `ctrl + h` (home) should also be disabled, but just wanted to clarify what your thoughts were.

Thanks agains

**Comment by ssleptsov** [comment]
Yeah, sure. It will disable/enable all hotkeys too based on that property
